package.json

```diff
@@ -13,6 +13,7 @@
     "url": "https://github.com/NativeScript/nativescript-hook.git"
   },
   "dependencies": {
+    "glob": "*",
```

Comment:
I've read your concern, but in case new version of glob has breaking change, the current code will stop working, so I prefer to set exact version (or at least one starting with `^`).